src/test/java/zippopotamus/steps/PostalCodeSteps.java

```diff
@@ -0,0 +1,41 @@
+package zippopotamus.steps;
+
+
+import io.cucumber.java8.En;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Assert;
+
+public class PostalCodeSteps implements En {
+
+    private Response response;
+    private String getLocationInformationByCountryAndPostalcode = "http://zippopotam.us/country/postalCode";
+    private String getLocationInformationByCountry_StateAndCity = "http://zippopotam.us/country/state/city";
+
+    public PostalCodeSteps() {
+        super();
+
+        Given("The user has access to the zippopotamus endpoint", () -> {
```

Comment:
We should redefine this step to do something significant, or just deleted it. 
If we want to keep this step anyway, we also could add some code to this, for example, we could print something like "This API does not need extra authentication"

Reply:
Done